cupyx/scipy/sparse/csr.py

```diff
@@ -139,23 +139,23 @@ def __mul__(self, other):
             elif other.ndim == 1:
                 self.sum_duplicates()
                 other = cupy.asfortranarray(other)
-                # csrmvEx does not work if nnz == 0
-                if self.nnz > 0 and cusparse.csrmvExIsAligned(self, other):
-                    for accelerator in _accelerator.get_routine_accelerators():
-                        if (accelerator == _accelerator.ACCELERATOR_CUB
-                                and other.flags.c_contiguous):
-                            return cub.device_csrmv(
-                                self.shape[0], self.shape[1], self.nnz,
-                                self.data, self.indptr, self.indices, other)
-                    return cusparse.csrmvEx(self, other)
+                for accelerator in _accelerator.get_routine_accelerators():
+                    if (accelerator == _accelerator.ACCELERATOR_CUB
+                            and other.flags.c_contiguous):
+                        return cub.device_csrmv(
+                            self.shape[0], self.shape[1], self.nnz,
+                            self.data, self.indptr, self.indices, other)
+                if (cusparse.check_availability('csrmvEx') and self.nnz > 0 and
+                        cusparse.csrmvExIsAligned(self, other)):
+                    # csrmvEx does not work if nnz == 0
+                    csrmv = cusparse.csrmvEx
+                if cusparse.check_availability('csrmv'):
```

Comment:
Sorry I just saw this PR. I think this needs to be changed to `elif`, otherwise it's overwritten.
```suggestion
                elif cusparse.check_availability('csrmv'):
```

Reply:
Ah, that's not good, I'll fix it right away.

Reply:
Thanks!